prepare/tasks/classification.py

```diff
@@ -3,8 +3,9 @@
 
 add_to_catalog(
     FormTask(
-        inputs=["text", "text_type", "class"],
-        outputs=["class", "label"],
+        inputs={"text": "str", "text_type": "str", "class": "str"},
+        outputs={"class": "str", "label": "str"},
```

Comment:
label should be List[str] in binary.  This is because it can be either be an empty list or a list with one element which is the class name.